addons/website/static/src/js/editor/snippets.options.js

```diff
@@ -1040,6 +1041,22 @@ options.registry.ThemeColors = options.registry.OptionsTab.extend({
 });
 
 options.registry.menu_data = options.Class.extend({
+    editMenuCallBack: function () {
```

Comment:
Missing doc + should be below the onFocus method, in a 
```javascript
//-------------------------------------
// Public
//---------------------------------------
```
section. But in fact it seems to be a private method so to start with `_`. Also why "callback" in the name?